plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/struct/DBStructUtils.java

```diff
@@ -399,7 +400,10 @@ public static String mapTargetDataType(DBSObject objectContainer, DBSTypedObject
                             }
                         }
                     } else if (targetType == null && dataKind == DBPDataKind.STRING) {
-                        if (typeNameLower.contains(TEXT_DATA_TYPE)) {
+                        if (typeNameLower.contains(TEXT_DATA_TYPE) || STRING_DATA_TYPE.equals(typeNameLower)) {
```

Comment:
We need something smarter

Reply:
Ok, I removed it. Let's just not use varchar without modifiers by default.